assets/js/base/components/payment-methods/payment-methods.js

```diff
@@ -70,7 +70,7 @@ const PaymentMethods = () => {
 		currentPaymentMethodInterface.current = paymentMethodInterface;
 	}, [ paymentMethods, paymentMethodInterface, activePaymentMethod ] );
 
-	const getRenderedTab = useCallback(
+	const getRenderedTab = useMemo(
 		() => ( selectedTab ) => {
```

Comment:
I think it's the same as:
```suggestion
	const getRenderedTab = useCallback(
		( selectedTab ) => {
```